add q1 report (#74)
aicoe-aiops/operate-first-data-science-community: docs/q1_report.md

> # Quick Glance at Q1

Reviewer: ```suggestion
# Operate First Data Science Community Meetup Analytics Report: Q1Y22
```

> Metrics on the Operate First Data Science Community Meetup between November 16, 2021 - March 31, 2022

Reviewer: ```suggestion
The following report outlines some key metrics and associated analysis of the community engagement for the Operate First Data Science Community Meetup between November 16, 2021 and March 31, 2022. If you would like to view recording of past meetups, or see the complete analytics dashboard yourself, please use the following to links: 
```

* [Detailed dashboard](https://datastudio.google.com/u/1/reporting/de85ebeb-c76a-4930-85bf-b07a1d2becd3/page/qiqpB/edit)
* [Past meetups and recordings](content.md)

### Aim of the community 

The Operate First Data Science Community Meetup is an event open to everyone interested in learning more about how various individuals, teams, and online communities are working towards democratizing the development of production-level AI. 

This meetup aims to embrace the multi-disciplinary nature of the current state of AI operations and cloud infrastructure by covering technical topics relevant to data scientists, software engineers, DevOps professionals, and statisticians alike. And all in an open public forum.

### Number of Meetups hosted 

8

### Total Number of Attendees

155

### Total Number of Unique Attendees

74

### Average number of Attendees per Meetup

28

### Internal vs external attendees

> 

Reviewer: Can you define what internal vs external means here? External to Red Hat? External to OPF?

Reviewer 2: plot shows attendees joining from outside of redhat.com domain, we can specify this

![image 1](assets/image1.png)

### Number of attendees for each meetup 

> **Consistent attendance across meetups with the lowest number of attendees being 21**

Reviewer: This was also the most recent meetup. Do you want to point out the downward trend, and not something like "We've noticed this downward trend in attendance over last few meetups. Suggestions for adressing this issue are outlined below" or something?

Author: Okay, sounds good. We will add that

![image 2](assets/image2.png)

### Speakers from various teams

**So far we have had **one external speaker** from the OSPO team present at the meetup**

![image 3](assets/image3.png)

> ## Post Meetup Engagement on YouTube

Reviewer: ```suggestion
## Meetup Engagement on YouTube
```

> So far we have 8 videos for the Operate First Data Science Community Meetup. 

Reviewer: ```suggestion
To date we have recorded and posted 8 videos of the Operate First Data Science Community Meetup to our YouTube channel. 
```

### Shares on Meetup Videos posted on Youtube

> **3 videos from the 8 operate first DS meetup videos land up in the top 10 shared videos across the channel**

Reviewer: ```suggestion
**3 videos from the 8 operate first DS meetup videos are in the top 10 shared videos across the channel**
```
Which 3? 

Also can you explain this graph a little more? What is a share?  What was the total number of shares?  Who is doing the sharing?

Author: Okay, we will add this.

> ![image 4](assets/image4.png)

Reviewer 3: For this image and the next one, is there some way to "highlight" which talks are from the meetup? It might not be obvious to some of the audience.

> ### Youtube Channel Subscribers obtained through posted Meetups

Reviewer: ```suggestion
### Youtube Channel Subscribers obtained through Meetup videos.
```

> **Operate First Community Meetup Videos accounted for over 30% of the channel’s subscribers**

Reviewer: Does this mean that between Nov and now, 30% of all chanel subscribes clicked "subscribe" while watch one of the meet up videos? How many is 30%?

> ![image 5](assets/image5.png)

Reviewer: Can you add a legend to this graph?

Author: Sure

> ### Views meetup videos on Youtube

Reviewer: ```suggestion
### Views 
```

> **4th highest viewed video across the channel is Operate First Community Meetup #1 amounting to 193 views**

Reviewer: Can you add a graph filtering for just the meetup videos? 

Can we see a value like, # of views of video 1 week after the meetup?

Author: We will add the graph filtering for just the meetup videos.

![image 6](assets/image6.png)

## Expectations for Q2

> * We aim to expand this community to external contributors and teams, increase participation from external speakers and attendees.

Reviewer: How do we plane to do this? Increase out reach?

> **4th highest viewed video across the channel is Operate First Community Meetup #1 amounting to 193 views**

Reviewer 4: "#1" links to the first pull request in the markdown, I guess because that's how github works.

Author: Thanks, fixed this!

# Operate First Data Science Community Meetup Analytics Report: Q1Y22

The following report outlines some key metrics and associated analysis of the community engagement for the Operate First Data Science Community Meetup between November 16, 2021 and March 31, 2022. If you would like to view recording of past meetups, or see the complete analytics dashboard yourself, please use the following to links: 

> * [Detailed dashboard](https://datastudio.google.com/u/1/reporting/de85ebeb-c76a-4930-85bf-b07a1d2becd3/page/qiqpB/edit)

Reviewer 2: Like google docs, this link will need you to sign into gmail to view. It is right now set to 
<img width="691" alt="image" src="https://user-images.githubusercontent.com/32435206/163230014-5a782830-d309-42d6-a6b3-17f8f09cd3c0.png">
External members will need to be specifically added. We do not have non-redhat stakeholders currently, so I think we should be fine?

> * [Detailed dashboard](https://datastudio.google.com/u/1/reporting/de85ebeb-c76a-4930-85bf-b07a1d2becd3/page/qiqpB/edit)

Reviewer 3: Sounds good! Just wanted to confirm :)
